dev-tools/notice/overrides.json

```diff
@@ -11,3 +11,14 @@
 {"name": "github.com/munnerz/goautoneg", "licenceType": "BSD-3-Clause"}
 {"name": "github.com/pelletier/go-buffruneio", "licenceType": "MIT"}
 {"name": "github.com/urso/magetools", "licenceType": "Apache-2.0"}
+{"name": "github.com/ajstarks/svgo", "licenceType": "CC-BY-4.0"}
+{"name": "github.com/bmizerany/pat", "licenceType": "MIT"}
+{"name": "github.com/cactus/go-statsd-client/statsd", "licenceType": "MIT"}
+{"name": "github.com/dnaeon/go-vcr", "licenceType":  "BSD"}
+{"name": "github.com/golang/freetype", "licenceType":  "GPL-3.0"}
+{"name": "github.com/hudl/fargo", "licenceType":  "MIT"}
+{"name": "github.com/influxdata/line-protocol", "licenceType":  "MIT"}
+{"name": "github.com/lightstep/lightstep-tracer-common/golang/gogo", "licenceType":  "MIT"}
+{"name": "github.com/pascaldekloe/goe", "licenceType":  "CC0-1.0"}
+{"name": "github.com/streadway/handy", "licenceType": "BSD-2-Clause"}
+{"name": "gonum.org/v1/plot", "licenceType": "BSD-3-Clause"}
```

Comment:
Interesting that the license checker cannot identify all these licenses :thinking: